packages/codegen/src/cli.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 Cognite AS
+import yargs, { CommandModule, Options } from 'yargs';
+import { createConfiguration } from './configuration';
+import { generateTypes } from './generate';
+import { updateSnapshot } from './snapshot';
+
+const packageOptions: Options = {
+  required: true,
+  describe: 'SDK package (e.g. stable, beta)',
+  type: 'string',
+};
+
+const fetchLatestCommand: CommandModule = {
+  command: 'fetch-latest',
+  describe:
+    'Fetch latest published Cognite OpenAPI document. Note that you will also need to run the command to generate types afterwards',
+  builder: (yargs) => yargs.option('package', packageOptions),
+  handler: async (argv) => {
+    await updateSnapshot({
+      package: argv.package as string,
+    });
+  },
+};
+
+const generateTypesCommand: CommandModule = {
+  command: 'generate-types',
+  describe: 'Generate types for configured services',
+  builder: (yargs) => yargs.option('package', packageOptions),
+  handler: async (argv) => {
+    await generateTypes({
+      package: argv.package as string,
+    });
+  },
+};
+
+const configureCommand: CommandModule = {
+  command: 'configure',
+  describe: 'Create configuration file to enable type generation for a service',
```

Comment:
"package or service"

Reply:
I suggest we just remove this. See https://github.com/cognitedata/cognite-sdk-js/pull/821